src/cpu/x64/jit_avx512_core_x8s8s32x_deconvolution.hpp

```diff
@@ -116,6 +130,13 @@ struct jit_avx512_core_x8s8s32x_deconv_fwd_kernel : public jit_generator {
     zmm_t zmm_bias = zmm_t(31);
     zmm_t zmm_prev_dst = zmm_t(31);
 
+    /* post ops */
+    const Xbyak::Reg64 reg_d_weights = r15;
+    const Xbyak::Reg64 reg_d_bias = r13;
```

Comment:
```suggestion
    const Xbyak::Reg64 reg_d_weights = aux_reg_filt_d;
    const Xbyak::Reg64 reg_d_bias = aux_reg_src_d;
```
NIT. So we can see that some registers are reused. Here and in same places in other changed (or added) kernels.

Reply:
Thanks for the comment, but I decided to leave it as before, because in my personal experience it is easier for me to see with this approach which registers I am using.